ptsprojects/stack.py

```diff
@@ -317,6 +317,47 @@ def __init__(self, uuid, oob, output_size, output_actions, input_size,
         # Node Identity
         self.proxy_identity = False
 
+        # MMDL status data
+        self.status_data = Property({
+            "Ack": True,
+            'Status': [],
+        })
+
+        self.rcv_status_data = Property({
```

Comment:
please add a comment explaining what's the difference between rcv_status_data and status_data here